src/thunderbots/software/param_server/cfg/Params.cfg

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env python
+import roslib
+PARAMS_CONFIG_NAMESPACE = "param_server"
+NODE_NAME = "param_server"
+from dynamic_reconfigure.parameter_generator_catkin import *
+
+"""
+This file contains the parameters used by ai, the current implementation requires 
+the parameter to be defined in this file, as well as in dynamic_parameters.cpp
+with the same parameter name
+
+The default values in dynamic_parameters.cpp are overwritten by the values defined
+here at runtime, the value provided in dynamic_parameters will only be used when 
+the parameter is missing from the cfg file
+"""
+
+gen = ParameterGenerator() # create the parameter generator
+gen.add("robot_expiry_buffer_milliseconds", int_t, 0, "brief description", 50, 0, 100)
+
+# NOTE: format gen.add(name, type, level, description, default_value, min, max)
+# min, max only apply for int/double types
+# for string and bool provide arguments up to and including default_value
+
+#######################################################################
+#                              NAVIGATOR                              #
+#######################################################################
+
+navigator_params = gen.add_group("Navigator")
+
+navigator_params.add("default_avoid_dist", double_t, 0, "brief description", 50,  0, 100)
+navigator_params.add("collision_avoid_velocity_scale", double_t, 0, "brief description", .5, 0,  1)
```

Comment:
> I'm a bit afraid that `brief description` here will forever remain unchanged. We should either change it to an actual brief description, or link a github issue here so that we know to update these at some point.

I think the above comment might have been missed in the last review, just putting it in this one to make it easier to manage for everyone.